assets/js/index.js

```diff
@@ -1,26 +1,173 @@
+      list2: 'Full Stack Dev',
+      list3: '2015',
+    },
+    image: 'static/imgs/Snapshoot Portfolio.png',
+    desc: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project3',
+  },
+  {
+    id: 4,
+    title: 'Uber',
+    list: {
+      list1: 'Uber',
+      list2: 'Lead Developer',
+      list3: '2015',
+    },
+    image: 'static/imgs/two.png',
+    desc: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project4',
+  },
+];
+
+// Add burger==========================================================================
+const burger = document.querySelector(
+  '.header__container__parhead__nav__ul__responsive',
+);
+const iwq = document.querySelector('.iwq');
+const parpopup = document.querySelector('.parpopup');
+const parpopuPopup = document.querySelector('.parpopup__popup');
 const navLinks = document.querySelectorAll('.mobile-nav-links');
 const email = document.querySelector('.form__control > input#email');
 const form = document.querySelector('form');
-const emailError = document.querySelector('#emailerror');
-
+const emailerror = document.querySelector('#emailerror');
+const pardetail = document.querySelector('.pardetail');
+// Add eventlistener===========================================================================
 burger.addEventListener('click', () => {
-  burger.classList.toggle('change');
-  mobileNav.classList.toggle('show');
+  if (burger.classList.contains('change')) {
+    burger.classList.remove('change');
+  } else {
+    burger.classList.add('change');
+  }
+
+  if (iwq.classList.contains('show')) {
+    iwq.classList.remove('show');
+  } else {
+    iwq.classList.add('show');
+  }
 });
+// Add function details============================================================================
+
+function detail(c) {
+  parpopup.style.display = 'flex';
+  const update = details.filter((item) => item.category === c);
+  parpopuPopup.innerHTML = `
+  <div class="head">
+  <span class="name">${update[0].title}</span>
+  <span id="close" onclick="closes()">&times;</span>
+</div>
+<ul>
+  <li><a href="">${update[0].list.list1}</a></li>
+  <li><a href="">${update[0].list.list2}</a></li>
+  <li><a href="">${update[0].list.list3}</a></li>
+</ul>
+<div class="popup__responsive">
+  <img src="${update[0].image}" alt="" class="responsive" />
+</div>
+<div class="types">
+  <div class="type">
+    <p>${update[0].desc}</p>
+  </div>
+  <div class="type">
+    <ul>
+      <li>${update[0].lang.lang1}</li>
+      <li>${update[0].lang.lang2}</li>
+      <li>${update[0].lang.lang3}</li>
+      <li>${update[0].lang.lang4}</li>
```

Comment:
- [ ] I noticed that only one of the `See Project` buttons is working and showing a popup, as shown in the attached video reference. Please ensure that when any of the `See Project` buttons is clicked, it opens up its corresponding popup window, as intended according to the design :100: 

[Screencast from 04-20-2023 09:23:45 AM.webm](https://user-images.githubusercontent.com/102699958/233321972-da45776c-9125-469c-a4a3-44af321f5687.webm)